front-end/src/pages/AuthorPage.tsx

```diff
@@ -123,7 +127,14 @@ export default function AuthorPage(props: any) {
           </Card>
         </Col>
         <Col>
-          {author && displayPosts()}
+          {author && props.loggedInUser && <PostList postEntries={postEntries} setPostEntries={setPostEntries} loggedInUser={props.loggedInUser} isResharable={true}/>}
+          
+          {author && isValidGithub(author.github) && 
+            <>
+              <h3>Github feed</h3>
+              <GithubFeed github={author.github}/>
+            </>
+          }
```

Comment:
If we're going with not showing author posts on their profile, we could have it so that our profile page just shows our GitHub feed since it's the only GitHub activity that we're able to see (I think). Otherwise, we should talk about where to put the GitHub feed. Could have another tab or navlink to our GitHub feed like we have the public and inbox feeds

Reply:
Good idea. I'm planning to add this in my next PR when I work on those related changes